### PR TITLE
New package: pwsafe (Password Safe)

### PR DESCRIPTION
**Pull request by pjones**
This pull request includes a new package for the open source tool Password Safe.

### CODE REVIEW

**Comment by edwtjo** [comment]
Please squash your commits into one

**Comment by pjones** [comment]
Sorry about that.  Commits are now squashed.

File: pkgs/applications/misc/pwsafe/default.nix [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{ stdenv, fetchurl, wxGTK, libuuid, xercesc, zip,
+  libXt, libXtst, libXi, xextproto, gettext, perl,
```
Comment (edwtjo) [inline pkgs/applications/misc/pwsafe/default.nix]: New lines should begin with a comma in order to align with the curly, remove the last comma above. You can fill each line to 72 chars.

File: pkgs/applications/misc/pwsafe/default.nix [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      LICENSE install/copyright $out/share/doc/passwordsafe
+
+    cp src/ui/wxWidgets/GCCUnicodeRelease/pwsafe $out/bin/
+    cp install/graphics/pwsafe.png $out/share/icons/hicolor/48x48/apps
+    cp docs/pwsafe.1 $out/share/man/man1
+    cp xml/* $out/share/pwsafe/xml
+    #  */
+  '';
+
+  meta = {
```
Comment (edwtjo) [inline pkgs/applications/misc/pwsafe/default.nix]: Reduce typing by using `meta = with stdenv.lib; {` and remove that prefix from the rhs assignments below.

**Comment by pjones** [comment]
Reformatted.  Thank you for pointing out those issues.

**Comment by edwtjo** [comment]
No problem, thank you for the contribution and welcome aboard :)

**Pull request merged**